src/main/java/org/thoughtcrime/securesms/WelcomeActivity.java

```diff
@@ -369,6 +369,7 @@ else if (progress<1000/*progress in permille*/) {
             else if (progress==1000/*done*/) {
                 DcHelper.getAccounts(this).startIo();
                 progressSuccess();
+                dcContext.assumeMultiDevice();
```

Comment:
this might reset deletion config for backups that are not from chatmail accounts :thinking:

Reply:
that's true, but it would change only "at once" to "never/automatic", maybe okay and may improve things also for normal accounts.

but i agree, it would be better to do this for chatmail only, where there are no other options that can go wrong (one more reason for no days/weeks/months here, btw)

@link2xt when we check for `is_chatmail` - is that set accordingly even if there is no network after import? so, is it cached in the database?

Reply:
i added a commit that resets to "never/automatic" only for chatmail profiles, assuming `is_chatmail` is persisted (it is, otherwise, we would also have some settings availablility flickering :)